content/en/agent/faq/agent-5-sectigo-root-ca-rotation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Agent v5 - Certificate Authority Update (2025)
+further_reading:
+- link: "/agent/guide/upgrade_agent_fleet_automation"
+  tag: "Documentation"
+  text: "Upgrade Agent Fleet Automation"
+---
+
+## Overview
+
+In May 2026, Datadog will deploy SSL certificates signed by a new certificate authority (Sectigo Root CA). **If you are running Datadog Agent v5 (especially versions below 5.32.7)**, your agents may lose connectivity with Datadog due to SSL certificate verification failures.
```

Comment:
I don't have the context to know which version of "agents" you're talking about, but we should always capitalize "Agent" when talking about the Datadog Agent. All other "agents" can be lowercase. If you've already done this, awesome!